lib/util/types.ts

```diff
@@ -138,6 +138,14 @@ export namespace TaxjarTypes {
     street?: string
   }
 
+  export interface AddressParams {
+    country?: string,
```

Comment:
Spent a few minutes trying to find this new JS syntax I wasn't familiar with before realizing it was TypeScript 😀